src/test/java/integration/CollectionMethodsTest.java

```diff
@@ -464,4 +471,21 @@ void errorWhenSomeMatchedButNoneShould() {
         "%nExpected: none of elements to match [value==cat] predicate"));
   }
 
+  @Test
+  void shouldItemWithText() {
+    $$("#user-table tbody tr  td.firstname")
+      .shouldBe(itemWithText("Bob"));
+  }
+
+  @Test
+  void errorWhenItemWithTextNotMatchedButShouldBe() {
+    String expectedText = "Luis";
+    assertThatThrownBy(()  -> $$("#user-table tbody tr  td.firstname").shouldBe(itemWithText(expectedText)))
```

Comment:
Please use `shouldHave(itemWithText)` instead of `shouldBe(itemWithText)`.

Reply:
fixed